test/integrate/dubbo/go-client/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang
+
+MAINTAINER scottwangsxll@gmail.com
```

Comment:
what is it mean?

Reply:
Okay, I have modified it in the latest commit.